docs/admin/monitoring/notifications/index.md

```diff
@@ -261,6 +263,43 @@ Administrators can configure which delivery methods are used for each different
 You can find this page under
 `https://$CODER_ACCESS_URL/deployment/notifications?tab=events`.
 
+## Custom notifications
+
+Custom notifications let you send an ad‑hoc notification to yourself using the Coder CLI.
+These are useful for surfacing the result of long-running tasks or important state changes.
+At this time, custom notifications can only be sent to the user making the request.
+
+To send a custom notification, execute [`coder notifications custom <title> <message>`](../../../reference/cli/notifications_custom.md).
+
+<!-- TODO(ssncferreira): Update when sending custom notifications to multiple users/roles is supported.
+	 Explain deduplication behaviour for multiple users/roles.
+	 See: https://github.com/coder/coder/issues/19768
+-->
+**Note:** The recipient is always the requesting user as targeting other users or groups isn’t supported yet.
+
+### Examples
+
+- Send yourself a quick update:
+
+```shell
+coder templates push -y && coder notifications custom "Template push complete" "Template version uploaded."
+```
```

Comment:
In this short-form case, it would be more ergonomic to allow

```shell
coder templates push -y && coder notifications custom "Template version uploaded"
```